apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -114,9 +114,8 @@ export default class ScriptEditor extends React.Component {
       .removeAttr('selected', true);
   };
 
-  handleUmbrellaSelectChange = event => {
```

Comment:
I noticed that the state for curriculumUmbrella was not being used so I removed it.